cbdr/cortex
# AR-71: WYSIWYG: Text Formatting Options Broken

## Purpose: 
It seems the version of Redactor's inline styling had a bug when altering highlighted passages. So I fixed this by adding in the version of Redactor that came after the bug fix I found [here](https://craftcms.stackexchange.com/questions/8960/apply-inline-styles-to-selected-text-in-redactor). I also updated `angular-redactor` to the latest version and emailed [Imperavi](https://imperavi.com/redactor/) for account information(they should get back to us in two days). 

Also luckily the last version of **Redactor** on bower happened to be newer than our current version which also happened to be a few versions after the fix.  See here [bower-redactor](https://github.com/stones/bower-redactor/blob/master/redactor/redactor.js)

So inline styling works again. 

## JIRA:
 https://cb-content-enablement.atlassian.net/browse/AR-71


## Steps to Take On Prod

n/a
## Changes:
* Changes to setup
  * update bower

* Architectural changes
  * n/a

* Migrations
  * n/a

* Library changes
  * bumped `angular-redactor`
  * added `inline-styles` redactor plugin to `legacy/vendor`
  * used replaced redactor with a slightly newer version from bower

* Side effects
  * n/a

## Screenshots
* Before
n/a
* After
n/a

## QA Links:
http://web.cortex-8.development.c66.me/legacy


## How to Verify These Changes
* Specific pages to visit
  * `legacy#/posts/new/article`

* Steps to take
  * first go here: http://web.cortex-8.development.c66.me/legacy 
  * then go to posts and create new
  * start writing or paste in some text and then highlight some text and see if bold italic and strikethrough works

* Responsive considerations
  * n/a


## Relevant PRs/Dependencies:

n/a
## Additional Information
